lemmy_apub/src/user.rs

```diff
@@ -271,7 +271,7 @@ impl FromApub for UserForm {
       send_notifications_to_email: false,
       matrix_user_id: None,
       actor_id: Some(check_actor_domain(person, expected_domain)?),
-      bio,
+      bio: Some(bio),
```

Comment:
We might want to do the same checks for length and valid username in here. Or maybe not, cause it would prevent us from federating with slightly modified instances. Eg Mastodon doesnt apply the post length limit to remote posts.

Reply:
hrm... I'll add a TODO note in there pointing this out.